### PR TITLE
AWS testing script

### PR DESCRIPTION
**Pull request by shantamsultania**
This is my code for issue 
Create AWS Script Template's for AWS End to end testing 
Fixes: #858
hey @kaustubhgupta please review the PR
-shantam

### CODE REVIEW

**Comment by shantamsultania** [comment]
hey, @kaustubhgupta I have made one pull request for both issues #858 and #856. 
i hope you will find them up to the mark

**Comment by kaustubhgupta** [comment]
@shantamsultania Umm, if one PR can solve both issues then why you created two separate issues?

**Comment by kaustubhgupta** [comment]
@santushtisharma10 @antrikshmisri do you have any knowledge of AWS?

**Comment by shantamsultania** [comment]
hey @kaustubhgupta they both are of different issues 
1. issue one is related to creating a testing script for AWS EC2 with the help of which a person can check there code/ script is working or not this will help him/her with saving was aws endorsement as well as money 
for example 
let's say you have an AWS code that you want to try out which requires the creation of 100 ec2 (usually companies do ) now if you will do so in your AWS Console it will cost you the creation of 10 ec2 now if you are using my script it will be created as a dummy and thus your money is saved. 

2. is related to creating a script for AWS lambda which can be run on a live environment for a management purpose 
for example, let's say you are a company that has 20 - 100 ec2 in which many are dummy or clone ec2 if there are left you will be priced for all the ec2 with the help of my script you can reduce this 

in short, they both are not related or the same 
I made a single pull request as they both are related to AWS so it will be easy for you guys to evaluate in place of evaluating both the scripts again with the same tech stack that is AWS

I hope this solved your doubts feel free to ping me for the same

**Comment by kaustubhgupta** [comment]
> hey @kaustubhgupta they both are of different issues
> 
> 1. issue one is related to creating a testing script for AWS EC2 with the help of which a person can check there code/ script is working or not this will help him/her with saving was aws endorsement as well as money
>    for example
>    let's say you have an AWS code that you want to try out which requires the creation of 100 ec2 (usually companies do ) now if you will do so in your AWS Console it will cost you the creation of 10 ec2 now if you are using my script it will be created as a dummy and thus your money is saved.
> 2. is related to creating a script for AWS lambda which can be run on a live environment for a management purpose
>    for example, let's say you are a company that has 20 - 100 ec2 in which many are dummy or clone ec2 if there are left you will be priced for all the ec2 with the help of my script you can reduce this
> 
> in short, they both are not related or the same
> I made a single pull request as they both are related to AWS so it will be easy for you guys to evaluate in place of evaluating both the scripts again with the same tech stack that is AWS
> 
> I hope this solved your doubts feel free to ping me for the same my discord username is #shantam1230 or you can leave a message here and I can contact you for the same if you can provide your username

But it will not reward you 25 + 25 points but 25 only as you have made only one PR. I hope you're aware of this thing. Points are rewarded for every PR and not issue. If you're okay with this then we can link both issues to this PR

**Comment by shantamsultania** [comment]
> > hey @kaustubhgupta they both are of different issues
> > 
> > 1. issue one is related to creating a testing script for AWS EC2 with the help of which a person can check there code/ script is working or not this will help him/her with saving was aws endorsement as well as money
> >    for example
> >    let's say you have an AWS code that you want to try out which requires the creation of 100 ec2 (usually companies do ) now if you will do so in your AWS Console it will cost you the creation of 10 ec2 now if you are using my script it will be created as a dummy and thus your money is saved.
> > 2. is related to creating a script for AWS lambda which can be run on a live environment for a management purpose
> >    for example, let's say you are a company that has 20 - 100 ec2 in which many are dummy or clone ec2 if there are left you will be priced for all the ec2 with the help of my script you can reduce this
> > 
> > in short, they both are not related or the same
> > I made a single pull request as they both are related to AWS so it will be easy for you guys to evaluate in place of evaluating both the scripts again with the same tech stack that is AWS
> > I hope this solved your doubts feel free to ping me for the same my discord username is #shantam1230 or you can leave a message here and I can contact you for the same if you can provide your username
> 
> But it will not reward you 25 + 25 points but 25 only as you have made only one PR. I hope you're aware of this thing. Points are rewarded for every PR and not issue. If you're okay with this then we can link both issues to this PR


ummm do one thing is you are ok with the scripts just let me know after review don't merge it I will make a separate PR for one script (why to lose those 25 points right ?) just let me know for any changes if any also if you have any doubts if it ok with you I will make one more PR or 2 separate PRs for the same that why mine, as well as your time, will also not be wasted. I hope this works out for you and the team.

**Comment by kaustubhgupta** [comment]
> > > hey @kaustubhgupta they both are of different issues
> > > 
> > > 1. issue one is related to creating a testing script for AWS EC2 with the help of which a person can check there code/ script is working or not this will help him/her with saving was aws endorsement as well as money
> > >    for example
> > >    let's say you have an AWS code that you want to try out which requires the creation of 100 ec2 (usually companies do ) now if you will do so in your AWS Console it will cost you the creation of 10 ec2 now if you are using my script it will be created as a dummy and thus your money is saved.
> > > 2. is related to creating a script for AWS lambda which can be run on a live environment for a management purpose
> > >    for example, let's say you are a company that has 20 - 100 ec2 in which many are dummy or clone ec2 if there are left you will be priced for all the ec2 with the help of my script you can reduce this
> > > 
> > > in short, they both are not related or the same
> > > I made a single pull request as they both are related to AWS so it will be easy for you guys to evaluate in place of evaluating both the scripts again with the same tech stack that is AWS
> > > I hope this solved your doubts feel free to ping me for the same my discord username is #shantam1230 or you can leave a message here and I can contact you for the same if you can provide your username
> > 
> > 
> > But it will not reward you 25 + 25 points but 25 only as you have made only one PR. I hope you're aware of this thing. Points are rewarded for every PR and not issue. If you're okay with this then we can link both issues to this PR
> 
> Ummm do one thing if you are ok with the scripts just let me know after review don't merge it I will make a separate PR for one script (why lose those 25 points right ?) just let me know for any changes if any also if you have any doubts if it ok with you I will make one more PR or 2 separate PRs for the same that why mine, as well as your time, will also not be wasted. I hope this works out for you and the team.

That's why I asked 😄. I think you should segregate the code first so that we can review it one by one more easily (More the code, more time required) 😄

**Comment by shantamsultania** [comment]
> > > > hey @kaustubhgupta they both are of different issues
> > > > 
> > > > 1. issue one is related to creating a testing script for AWS EC2 with the help of which a person can check there code/ script is working or not this will help him/her with saving was aws endorsement as well as money
> > > >    for example
> > > >    let's say you have an AWS code that you want to try out which requires the creation of 100 ec2 (usually companies do ) now if you will do so in your AWS Console it will cost you the creation of 10 ec2 now if you are using my script it will be created as a dummy and thus your money is saved.
> > > > 2. is related to creating a script for AWS lambda which can be run on a live environment for a management purpose
> > > >    for example, let's say you are a company that has 20 - 100 ec2 in which many are dummy or clone ec2 if there are left you will be priced for all the ec2 with the help of my script you can reduce this
> > > > 
> > > > in short, they both are not related or the same
> > > > I made a single pull request as they both are related to AWS so it will be easy for you guys to evaluate in place of evaluating both the scripts again with the same tech stack that is AWS
> > > > I hope this solved your doubts feel free to ping me for the same my discord username is #shantam1230 or you can leave a message here and I can contact you for the same if you can provide your username
> > > 
> > > 
> > > But it will not reward you 25 + 25 points but 25 only as you have made only one PR. I hope you're aware of this thing. Points are rewarded for every PR and not issue. If you're okay with this then we can link both issues to this PR
> > 
> > 
> > Ummm do one thing if you are ok with the scripts just let me know after review don't merge it I will make a separate PR for one script (why lose those 25 points right ?) just let me know for any changes if any also if you have any doubts if it ok with you I will make one more PR or 2 separate PRs for the same that why mine, as well as your time, will also not be wasted. I hope this works out for you and the team.
> 
> That's why I asked 😄. I think you should segregate the code first so that we can review it one by one more easily (More the code, more time required) 😄

ok let's delete this PR and I will make 2 separate for both issues just let me know you have understood the whole initiative of scripts then will delete this PR

**Comment by shantamsultania** [comment]
I am creating 2 new PRs then
